### PR TITLE
Fix problem: datastore encodes metrics data into json string again while saving

### PR DESCRIPTION
**Pull request by chicm-ms**
Previously datastore encodes metrics data into json string before saving it into database because the field is of string type.
Later the metrics data field is already encoded into json string by SDK, then datastore should remove the encoding code.
This problem causes the metric data look like this:
```
 "finalMetricData":[
            {
                "data":"\"{\\\"default\\\": 0.6, \\\"loss\\\": 0.66, \\\"other\\\": 0.666}\"",
                "parameterId":"0",
                "sequence":0,
                "timestamp":1597147814203,
                "trialJobId":"ABRiy",
                "type":"FINAL"
            }
        ],
```

The correct result should be this:
```
"finalMetricData":[
            {
                "data":"{\"default\": 0.6, \"loss\": 0.66, \"other\": 0.666}",
                "parameterId":"0",
                "sequence":0,
                "timestamp":1597148506323,
                "trialJobId":"epX8Z",
                "type":"FINAL"
            }
        ],
```

TODO:
1. nnictl, nnicli -- corresponding update

### CODE REVIEW

**Comment by SparkSnail** [comment]
Tested on current nnictl, don't have affect for nnictl.

File: src/nni_manager/core/sqlDatabase.ts [hunk 1]
```diff
@@ -203,7 +203,7 @@ class SqlDB implements Database {
     public storeMetricData(trialJobId: string, data: string): Promise<void> {
         const sql: string = 'insert into MetricData values (?,?,?,?,?,?)';
         const json: MetricDataRecord = JSON.parse(data);
-        const args: any[] = [Date.now(), json.trialJobId, json.parameterId, json.type, json.sequence, JSON.stringify(json.data)];
+        const args: any[] = [Date.now(), json.trialJobId, json.parameterId, json.type, json.sequence, json.data];
```
Comment (QuanluZhang) [inline src/nni_manager/core/sqlDatabase.ts:206]: maybe we should remove JSON.parse in this line https://github.com/microsoft/nni/blob/bcefce6a814c2c1228e25fcd98c88fea9e8111a9/src/nni_manager/core/nniDataStore.ts#L170?

Comment (QuanluZhang) [reply src/nni_manager/core/sqlDatabase.ts:206]: BTW, do we have unit test on data export and resume? maybe we should run IT for this pr

**Comment by ultmaster** [comment]
This PR will break:

* Compatibility with experiments from previous releases
* WebUI mock data needs to be converted

I think it's too risky to merge this PR at this time before code freeze. Suggest merging into master after cutting branch.

File: src/nni_manager/core/sqlDatabase.ts (hunk 1 above)
Comment (chicm-ms) [reply src/nni_manager/core/sqlDatabase.ts:206]: Yes, we need data import/export IT test case.

**Pull request closed without merging**